### ISSUE TITLE
the default parameter for 'beta' in YoloNASPoseTaskAlignedAssigner

### ISSUE THREAD

**Issue by ZYX-MLer** [Deci-AI/super-gradients#1800]
### 💡 Your Question

I trained a hand keypoint detection algorithm using the Coco database and the YOLO-NAS-Pose architecture. 

During experimentation, I observed that the default parameter for 'beta' in YoloNASPoseTaskAlignedAssigner is set to 6. In the training process, I found that the loss calculation involves the term 
`'alignment_metrics = bbox_cls_scores.pow(self.alpha) * ious.pow(self.beta),' `
where 'ious' is a value between 0 and 1. 
Throughout training, I noticed that the calculated loss, when weighted with 'alignment_metrics,' resulted in a very small final loss, maybe 1e-6. 
I observed that the coco2017_yolo_nas_pose_train_params.yaml file does not redefine this parameter. Is it normal to set self.beta to 6? 
Additionally, are there any specific considerations or precautions when using NAS-Pose to train other models? 

Thank you.

**Comment by ZYX-MLer** [comment]
the first and second loss is as follows:

SUMMARY OF EPOCH 0
├── Train
│   ├── Yolonasposeloss/loss_cls = 0.0371
│   ├── Yolonasposeloss/loss_iou = 0.0316
│   ├── Yolonasposeloss/loss_dfl = 0.0008
│   ├── Yolonasposeloss/loss_pose_cls = 0.0092
│   ├── Yolonasposeloss/loss_pose_reg = 0.8098
│   └── Yolonasposeloss/loss = 0.8885
└── Validation
    ├── Yolonasposeloss/loss_cls = 0.0012
    ├── Yolonasposeloss/loss_iou = 0.0157
    ├── Yolonasposeloss/loss_dfl = 0.0003
    ├── Yolonasposeloss/loss_pose_cls = 0.0008
    ├── Yolonasposeloss/loss_pose_reg = 0.373
    ├── Yolonasposeloss/loss = 0.3911
    ├── Ap = 0.0
    └── Ar = 0.0



SUMMARY OF EPOCH 1
├── Train
│   ├── Yolonasposeloss/loss_cls = 0.0006
│   │   ├── Epoch N-1      = 0.0371 (↘ -0.0365)
│   │   └── Best until now = 0.0371 (↘ -0.0365)
│   ├── Yolonasposeloss/loss_iou = 0.0139
│   │   ├── Epoch N-1      = 0.0316 (↘ -0.0177)
│   │   └── Best until now = 0.0316 (↘ -0.0177)
│   ├── Yolonasposeloss/loss_dfl = 0.0004
│   │   ├── Epoch N-1      = 0.0008 (↘ -0.0004)
│   │   └── Best until now = 0.0008 (↘ -0.0004)
│   ├── Yolonasposeloss/loss_pose_cls = 0.0014
│   │   ├── Epoch N-1      = 0.0092 (↘ -0.0078)
│   │   └── Best until now = 0.0092 (↘ -0.0078)
│   ├── Yolonasposeloss/loss_pose_reg = 0.3772
│   │   ├── Epoch N-1      = 0.8098 (↘ -0.4326)
│   │   └── Best until now = 0.8098 (↘ -0.4326)
│   └── Yolonasposeloss/loss = 0.3934
│       ├── Epoch N-1      = 0.8885 (↘ -0.4951)
│       └── Best until now = 0.8885 (↘ -0.4951)
└── Validation
    ├── Yolonasposeloss/loss_cls = 0.001
    │   ├── Epoch N-1      = 0.0012 (↘ -0.0003)
    │   └── Best until now = 0.0012 (↘ -0.0003)
    ├── Yolonasposeloss/loss_iou = 0.0136
    │   ├── Epoch N-1      = 0.0157 (↘ -0.0021)
    │   └── Best until now = 0.0157 (↘ -0.0021)
    ├── Yolonasposeloss/loss_dfl = 0.0003
    │   ├── Epoch N-1      = 0.0003 (↘ -0.0)
    │   └── Best until now = 0.0003 (↘ -0.0)
    ├── Yolonasposeloss/loss_pose_cls = 0.0008
    │   ├── Epoch N-1      = 0.0008 (↘ -1e-04)
    │   └── Best until now = 0.0008 (↘ -1e-04)
    ├── Yolonasposeloss/loss_pose_reg = 0.3659
    │   ├── Epoch N-1      = 0.373  (↘ -0.0071)
    │   └── Best until now = 0.373  (↘ -0.0071)
    ├── Yolonasposeloss/loss = 0.3816
    │   ├── Epoch N-1      = 0.3911 (↘ -0.0096)
    │   └── Best until now = 0.3911 (↘ -0.0096)
    ├── Ap = 0.0
    │   ├── Epoch N-1      = 0.0    (= 0.0)
    │   └── Best until now = 0.0    (= 0.0)
    └── Ar = 0.0
        ├── Epoch N-1      = 0.0    (= 0.0)
        └── Best until now = 0.0    (= 0.0)

**Comment by ZYX-MLer** [comment]
in the 8th epoch, the loss is more less, but the ap and ar is still zero


SUMMARY OF EPOCH 8
├── Train
│   ├── Yolonasposeloss/loss_cls = 1e-04
│   │   ├── Epoch N-1      = 1e-04  (↗ 0.0)
│   │   └── Best until now = 1e-04  (↗ 0.0)
│   ├── Yolonasposeloss/loss_iou = 0.0043
│   │   ├── Epoch N-1      = 0.0047 (↘ -0.0004)
│   │   └── Best until now = 0.0047 (↘ -0.0004)
│   ├── Yolonasposeloss/loss_dfl = 1e-04
│   │   ├── Epoch N-1      = 1e-04  (↗ 0.0)
│   │   └── Best until now = 1e-04  (↗ 0.0)
│   ├── Yolonasposeloss/loss_pose_cls = 0.0003
│   │   ├── Epoch N-1      = 0.0003 (↘ -0.0)
│   │   └── Best until now = 0.0003 (↘ -0.0)
│   ├── Yolonasposeloss/loss_pose_reg = 0.1301
│   │   ├── Epoch N-1      = 0.1355 (↘ -0.0054)
│   │   └── Best until now = 0.1355 (↘ -0.0054)
│   └── Yolonasposeloss/loss = 0.135
│       ├── Epoch N-1      = 0.1408 (↘ -0.0058)
│       └── Best until now = 0.1408 (↘ -0.0058)
└── Validation
    ├── Yolonasposeloss/loss_cls = 0.0002
    │   ├── Epoch N-1      = 0.0005 (↘ -0.0003)
    │   └── Best until now = 0.0003 (↘ -1e-04)
    ├── Yolonasposeloss/loss_iou = 0.0086
    │   ├── Epoch N-1      = 0.0102 (↘ -0.0016)
    │   └── Best until now = 0.0066 (↗ 0.002)
    ├── Yolonasposeloss/loss_dfl = 0.0002
    │   ├── Epoch N-1      = 0.0003 (↘ -0.0)
    │   └── Best until now = 0.0002 (↗ 1e-04)
    ├── Yolonasposeloss/loss_pose_cls = 0.0005
    │   ├── Epoch N-1      = 0.0004 (↗ 1e-04)
    │   └── Best until now = 0.0004 (↗ 1e-04)
    ├── Yolonasposeloss/loss_pose_reg = 0.2591
    │   ├── Epoch N-1      = 0.3269 (↘ -0.0678)
    │   └── Best until now = 0.1824 (↗ 0.0767)
    ├── Yolonasposeloss/loss = 0.2686
    │   ├── Epoch N-1      = 0.3382 (↘ -0.0696)
    │   └── Best until now = 0.1906 (↗ 0.078)
    ├── Ap = 0.0
    │   ├── Epoch N-1      = 0.0    (= 0.0)
    │   └── Best until now = 0.0    (= 0.0)
    └── Ar = 0.0
        ├── Epoch N-1      = 0.0    (= 0.0)
        └── Best until now = 0.0    (= 0.0)
